src/source-nfq.c

```diff
@@ -536,9 +536,11 @@ static int NFQCallBack(struct nfq_q_handle *qh, struct nfgenmsg *nfmsg,
     if (ret == -1) {
 #ifdef COUNTERS
         NFQQueueVars *q = NFQGetQueue(ntv->nfq_index);
```

Comment:
`NFQGetQueue` should probably not be able to return NULL instead, adding some asserts. The checks that are done in the function can probably move to some startup routine.

Reply:
@victorjulien I agree but that might be outside the scope of this PR. Perhaps a tracking issue for removing the possibility of a `NULL` return value would be a reasonable option?

Reply:
I don't see it as out of scope. I don't see the point of adding runtime complexity for a very theoretical issue, essentially just to satisfy a code checker we have no access to.

Reply:
I agree that it's theoretical issue, but the probability exists, may be NFQGetQueue() must not be able to return a null pointer?